include/kernels/ConcreteLatentHeat.h

```diff
@@ -17,14 +17,11 @@
 #include "TimeDerivative.h"
 
 // Forward Declaration
```

Comment:
Here's another comment that no longer applies since we're not forward declaring anything.